opslevel/datasource_opslevel_property_definition.go

```diff
@@ -10,11 +10,20 @@ func datasourcePropertyDefinition() *schema.Resource {
 	return &schema.Resource{
 		Read: wrap(datasourcePropertyDefinitionRead),
 		Schema: map[string]*schema.Schema{
+			"identifier": {
+				Type:        schema.TypeString,
+				Description: "The id or alias of the property definition to find.",
+				Required:    true,
+			},
 			"id": {
```

Comment:
Should `"id"` still be included in this schema since `"identifier"` was added?

Reply:
You're right, they (id + alias) should be removed after adding `identifier` just like in the repository datasource that uses an `identifier`. Changed!